src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -263,7 +182,7 @@ static bool            segv_handler_inited;
 struct sigaction       old_segv;
 
 /* the flag to indicate whether initlization is finished or not */
-static bool            hook_enabled;
+bool                   hook_enabled;
```

Comment:
If this is now public then it (and any other exported symbols) needs a `d_` prefix or similar.

Reply:
I will fix them in next commit. Thank you!

Reply:
Variables are renamed as suggested. Thank you!